CONTRIBUTOR-DOCS/05_component-info/README.md

```diff
@@ -0,0 +1,21 @@
+<!-- Generated breadcrumbs - DO NOT EDIT -->
+
+[CONTRIBUTOR-DOCS](../README.md) / Component Info
+
+<!-- Document title (editable) -->
+
+# Component Info
+
+<!-- Generated TOC - DO NOT EDIT -->
+
+<details open>
+<summary><strong>Beneath this doc</strong></summary>
+
+- Badge
+- Progress Circle
```

Comment:
This will be where the markdown files from component analysis docs (#5770) go eventually, right? Would this file get integrated with the [README](https://github.com/adobe/spectrum-web-components/blob/2nd-gen-component-analysis/migration-roadmap/README.md) from there? And then over here we'd link to each component and potentially also the prompt?

Reply:
thats how i was interpreting that section